YYImage/YYAnimatedImageView.m

```diff
@@ -532,6 +532,11 @@ - (void)step:(CADisplayLink *)link {
 - (void)displayLayer:(CALayer *)layer {
     if (_curFrame) {
         layer.contents = (__bridge id)_curFrame.CGImage;
+    } else {
+        // If we have no animation frames, call super implementation. iOS 14+ UIImageView use this delegate method for rendering.
+        if ([UIImageView instancesRespondToSelector:@selector(displayLayer:)]) {
+            [super displayLayer:layer];
```

Comment:
This fix comes from a public GH fork of YYImage, which is moribund. Maybe we should move away from YYImage?

Reply:
Fix from: sundayfun@df226ee